packages/generate-password/src/index.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * Forked from https://github.com/WordPress/wporg-two-factor/blob/5241afeac49739ba58d8162f468434291dbc0fb6/settings/src/components/password.js#L132-L160
```

Comment:
```suggestion
 * Forked from https://github.com/WordPress/wporg-two-factor/blob/3e19f8a11e9a18633a77a3f7a36f0ef9a984b3d3/settings/src/components/password.js#L136-L164
```

I've merged https://github.com/WordPress/wporg-two-factor/pull/33/, so `5241afeac49739ba58d8162f468434291dbc0fb6` is now on a deleted branch that GH will probably clean up eventually. `3e19f8a11e9a18633a77a3f7a36f0ef9a984b3d3` is in `trunk` and should be permanent.

Reply:
Thanks! I updated the link in https://github.com/Automattic/wp-calypso/pull/71704/commits/29e2ad21caa190be3aa7ec67f08ef834d61e0b03

Reply:
It looks like that references `trunk`, which isn't a permalink. The line number refs will point to different lines soon. Referencing the specific commit also makes it obvious which commit this code was forked from.

`trunk` works too though if you prefer, but in that case I'd recommend removing the line numbers.